shared/local-debug.native.js

```diff
@@ -19,7 +19,7 @@ let config = {
   overrideLoggedInTab: null
 }
 
-if (__DEV__ && false) {
+if (__DEV__ && true) {
```

Comment:
while we're devving this out maybe we should leave this on, then we get nicer actions logging